packages/core/client.js

```diff
@@ -139,11 +139,15 @@ class Client {
     const session = new Session()
 
     session.app = {
-      releaseStage: this._config.releaseStage,
-      version: this._config.appVersion,
-      type: this._config.appType
+      ...{
```

Comment:
What does this creation and spreading of an object literal achieve?

Reply:
ah this a mistake, I'll rework it